### PR TITLE
Setup mobile  skeleton

### PR DESCRIPTION
**Pull request by nimplay**
In this PR I:
- Used the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- Built the  2 following sections on my project:
    - the toolbar (or header):
    - the headline section (right after the header):
- Used Flexbox in all 2 sections inorder to lay out the elements on the page.
- Stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

**Comment by nimplay** [comment]
summary:
fix all formatting details according to sigma and change branch name and PR. besides fixing the linters and changing the necessary tags for the project.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            <img src="./Images/hand.png" alt="">
+          </li>
+          <li class="socialimg">
+            <img src="./Images/Icon-Twitter.png" alt="">
+          </li>
+          <li class="socialimg">
+            <img src="./Images/Icon-Medium.png" alt="">
+          </li>
+        </ul>                      
+  </section>
```
Comment (ShahierNashaat) [inline index.html:49]: - [ ] Please make your design like the Figma design as you need to remove the spaces from the right, the left, and the bottom, also remove the space at the top of the header.
Your Design | Figma Design
---------------- | -------------------
Screenshot from 2022-05-26 23-29-15 | Screenshot from 2022-05-26 23-02-35

- [ ] Kindly ignore the `N`, `A`, and `O` icons.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @nimplay , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [Optional] Please next time when you get the required changes in a pull request don't close it as the second reviewer needs to know if you did the required changes from the first reviewer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (nimplay) [reply index.html:49]: The leters n, a and d are not part of the layout, they are comment buttons...

Comment (ShahierNashaat) [reply index.html:49]: Yes I know I have mentioned that

**Review by Kakalanp — APPROVED** [review]
# STATUS: APPROVED :green_circle: 

### Your project is approved @nimplay! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions

- You might want to improve your README to make your project look more professional :briefcase: it's just the details, but you could have that as a requirement in the future :100: 

- This weird looking folder might not look like a big deal, but it is a folder from another repository that is probably there by accident, consider deleting it :wink: 
image


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**